unitex/src/fr/umlv/unitex/frames/GraphPathFrame.java

```diff
@@ -426,6 +427,9 @@ private void runButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIR
         if ( !checkLoopsCheckbox.isSelected() ) {
                 cmd = cmd.noLoopCheck();
         }
+        if (ConfigManager.getManager().isKorean(null)) {
+			cmd = cmd.korean();
```

Comment:
Make sure to use the right indentation.

Reply:
Done.
But I had a problem when I wanted to push the changes, I got this message (I don't know why) : 
! [rejected]          korean -> korean (non-fast-forward)
I resolved my problem but it made 3 commits.